app/services/Capi.scala

```diff
@@ -20,8 +20,6 @@ import scala.concurrent.{ExecutionContext, Future, Promise}
 class GuardianCapi(config: ApplicationConfiguration)(implicit ex: ExecutionContext)
   extends GuardianContentClient(apiKey = config.contentApi.editionsKey) with Capi with Logging {
 
-  private def prefillHelper = PrefillHelper(EditionsTemplates.templates)
```

Comment:
it is used in `EditionsTemplating` as well , the `defineContentQueryTimeWindow` function

Reply:
`Capi` is used only for prefills here so i would put that into prefill package as well and have some main prefill service or prefill helper that have the main public functions like `getPrefillArticles`

Now it may be a bit confusing

We generate prefills while 2 requests: `EditionsController.createIssue` and `EditionsController.getPrefillForCollection` 

`EditionsController.createIssue` call `generateEditionTemplate` which call `capi.getUnsortedPrefillArticleItems`

`EditionsController.getPrefillForCollection`  call `capi.getPrefillArticles`

Reply:
i am keen to here your suggestions

Reply:
added changes related to our conversation